src/test/functional/features/joint-check-your-answers.feature

```diff
@@ -1,3 +1,4 @@
+@flaky
```

Comment:
You should be able to merge in master and remove this flaky tag, as this should be fixed by the new `Given I create a new user and login` step :tada: